docs/design/design.md

```diff
@@ -135,56 +192,82 @@ Other flags that are set:
  - `--requestheader-client-ca-file` to `front-proxy-ca.crt`
  - `--admission-control` to `NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota`
    - ...or whatever the recommended set of admission controllers is at a given version
- - `--storage-backend` to `etcd3`. Support for `etcd2` in kubeadm is dropped.
  - `--kubelet-preferred-address-types` to `InternalIP,ExternalIP,Hostname`
    - This makes `kubectl logs` and other apiserver -> kubelet communication work in environments where the hostnames of the nodes aren't resolvable
- - `--requestheader-username-headers=X-Remote-User`, `--requestheader-group-headers=X-Remote-Group`, `--requestheader-extra-headers-prefix=X-Remote-Extra-`, --requestheader-allowed-names=front-proxy-client` so the front proxy (API Aggregation) communication is secure.
+ - `--requestheader-username-headers=X-Remote-User`, `--requestheader-group-headers=X-Remote-Group`, `--requestheader-extra-headers-prefix=X-Remote-Extra-`, `--requestheader-allowed-names=front-proxy-client` so the front proxy (API Aggregation) communication is secure.
```

Comment:
Can you link the API aggregation proposal/design doc?

Reply:
We have more flags for this as well: `requestheader-client-ca-file`, `proxy-client-cert-file`, `proxy-client-key-file`

Reply:
Done, but not sure if this is the doc you are referring to...